manta-trusted-setup/src/groth16/mod.rs

```diff
@@ -19,8 +19,11 @@
 pub mod kzg;
 pub mod mpc;
 
-#[cfg(all(feature = "bincode", feature = "serde"))]
-#[cfg_attr(doc_cfg, doc(cfg(all(feature = "bincode", feature = "serde"))))]
+#[cfg(all(feature = "bincode", feature = "serde", feature = "tokio"))]
```

Comment:
Should we have `tokio` here or for binary only?

Reply:
I don't entirely understand why but removing `tokio` here makes the module `groth16::ceremony::client` to be unfindable by `groth16::ceremony::config::ppot`. The check `cargo clippy --no-default-features --features ppot-client` does not pass if we remove `tokio` here. Perhaps it could be moved somewhere else?